src/NHibernate/AdoNet/OracleDataClientBatchingBatcher.cs

```diff
@@ -125,21 +125,32 @@ protected override void DoExecuteBatch(IDbCommand ps)
 				// this value is not a part of the ADO.NET API.
 				// It's and ODP implementation, so it is being set by reflection
 				SetArrayBindCount(arraySize);
-				int rowsAffected;
 				try
 				{
-					rowsAffected = _currentBatch.ExecuteNonQuery();
+					int rowsAffected;
+					try
+					{
+						rowsAffected = _currentBatch.ExecuteNonQuery();
+					}
+					catch (DbException e)
+					{
+						// Message from DbException could be useful
+						var additionalMessage = string.IsNullOrEmpty(e.Message) ? "." : string.Concat(": ", e.Message);
```

Comment:
Yes, that's my point, it's already there. Based on the comment in the PR and also in the code, the additionalMessage seems to have been added based on the idea that without it, the message from the original exception was somehow lost. As far as I can tell, the inner exception will be there as expected and then for anything that does log the exception chain properly, this will just be unfortunate duplication. t's not a big issue and I don't care too much either way, but I think it should be removed.